compiler/passes/convert-typed-uast.cpp

```diff
@@ -4938,13 +4938,23 @@ Expr* TConverter::convertParenlessCallOrNull(const AstNode* node, RV& rv) {
   }
 
   auto calledFn = findOrConvertFunction(rf);
-  auto ret = new CallExpr(calledFn);
+  Expr* ret = nullptr;
 
   if (sig->isMethod()) {
-    TC_UNIMPL("Handle parenless method call!");
-    return TC_PLACEHOLDER(this);
+    if (node->isIdentifier()) {
+      INT_ASSERT(cur.fnSymbol && cur.fnSymbol->isMethod());
+      ret = new CallExpr(calledFn, cur.fnSymbol->_this);
```

Comment:
The `_this`-fetching code feels like it should be in a helper (David mentioned needing access to the receiver before, so there may even be other typed converter code that does variations of this code)